source/includes/entityref/paybill/_clientcorporationrateagreementcardline.md

```diff
@@ -30,35 +30,61 @@ An Entity representing each individual rate on a Client Corporation
         <tr class="odd">
             <td>alias</td>
             <td>String (100)</td>
-            <td></td>
+            <td>Name of Earn Code that will be displayed in Time and Expense and on the Invoice Statement</td>
             <td></td>
             <td></td>
         </tr>
         <tr class="even">
             <td>billCurrencyUnit</td>
             <td>To-one association</td>
-            <td>CurrencyUnit</td>
+            <td>Currency Unit for billing
+                <p>Default fields:</p>
+                <ul>
+                    <li>id</li>
+                    <li>alphabeticCode</li>
+                    <li>name</li>
+                </ul>                
+            </td>
             <td></td>
             <td></td>
         </tr>
         <tr class="odd">
             <td>billMultiplier</td>
             <td>BigDecimal</td>
-            <td></td>
+            <td>
+                <p>The multiplier between the REG billRate and the current line’s billRate.</p>
+                <ul>
+                    <li>REG billMultiplier is always 1 (read only)</li>
+                    <li>The following are defaulted from PayBillSetting:</li>
+                        <ul>
+                            <li>overtimeBillMultiplier (OT billMultiplier) - Set on ClientCorporationRateAgreementCardLine.billMultiplier for the OT earnCode</li>
+                            <li>doubleTimeBillMultiplier (DT billMultiplier) - Set on ClientCorporationRateAgreementCardLine.billMultiplier for the DT earnCode</li>
+                        </ul>
+                </ul>
+                <p>OT and DT billMultipliers get automatically calculated in code based off their respective billRate and REG billRate.</p>
+            </td>
             <td></td>
             <td></td>
         </tr>
         <tr class="even">
             <td>billRate</td>
             <td>BigDecimal</td>
-            <td></td>
+            <td>
+                <p>Rate for billing (stored as an hourly rate).</p>
+                <p>billRate can get automatically calculated in code based off its respective payRate and markupPercent.</p>
+            </td>
             <td></td>
             <td></td>
         </tr>
         <tr class="odd">
             <td>clientCorporationRateAgreementCardLineGroup</td>
             <td>To-one association</td>
-            <td>ClientCorporationRateAgreementCardLineGroup</td>
+            <td>ClientCorporationRateAgreementCardLineGroup
```

Comment:
Add period after ClientCorporationRateAgreementCardLineGroup for consistency with other descriptions.